spec/integ/crypto/rust-crypto.spec.ts

```diff
@@ -20,6 +20,12 @@ import fetchMock from "fetch-mock-jest";
 
 import { createClient, CryptoEvent, IndexedDBCryptoStore } from "../../../src";
 import { populateStore } from "../../test-utils/test_indexeddb_cryptostore_dump";
+import {
+    MIGRATION_BACKUP,
+    MIGRATION_KEY_QUERY_RESPONSE,
+    ROTATED_MIGRATION_KEY_QUERY_RESPONSE,
```

Comment:
these seem like they need better names. We have three sets of data, all for testing migration. It is very confusing to have an identifier called `MIGRATION_BACKUP` which is only relevant to one of those three sets of data.

Reply:
Refactored.

Reply:
This is much nicer now. Thank you!